packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -91,12 +91,12 @@ class ScrollBehavior {
     switch (getPlatform(context)) {
       case TargetPlatform.iOS:
       case TargetPlatform.macOS:
-        return _bouncingPhysics;
+        return const BouncingScrollPhysics();
```

Comment:
This is almost definitely going to break something - the change to use the RangeMaintainingScrollPhysics was made in https://github.com/flutter/flutter/pull/56521 (/cc @hixie) 

You can look at the tests in that PR to help determine what it was fixing.

This PR would need to have at least one test that explains what behavior it's fixing. I'll add some more comments on the issue you've linked.